backend/tracim_backend/lib/core/content.py

```diff
@@ -452,9 +457,41 @@ def create(
         content.revision_type = ActionDescription.CREATION
         content.content_namespace = content_namespace
 
+        if template_id:
+            template = self.get_one(template_id)
+
+            if label:
+                content = self.copy(template, new_label=label, copy_revision=False, do_save=False)
+            elif filename:
+                content = self.copy(
+                    template, new_label=filename, copy_revision=False, do_save=False
+                )
+
         if do_save:
             self._session.add(content)
             self.save(content, ActionDescription.CREATION, do_notify=do_notify)
+
+            if template_id:
+                tags_values = (
```

Comment:
Doesn't `TagLib` have a method to get all tags of a content? Instead of manually building a query here?
BTW this handling should be part of `copy()` as I think that copying a content should also copy its tags, check with the PO committee.

Reply:
The choice made was to create an issue and leave it "as it is" so we can merge this development.
See https://github.com/tracim/tracim/issues/5700